### ISSUE TITLE
TypeScript 1.8 support

### ISSUE THREAD

**Issue by ghpabs** [panuhorsmalahti/gulp-tslint#47]
Update support for TS 1.8.

```
npm ERR! peerinvalid The package typescript@1.8.0-dev.20151204 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer tslint@3.1.1 wants typescript@>=1.7.3
```

TypeScript 1.8 is one of my `devDependencies` as below:

`"typescript": "^1.8.0-dev.20151204"`

**Comment by panuhorsmalahti** [comment]
This peerDependency issue comes from tslint: https://github.com/palantir/tslint/blob/master/package.json#L43

I suggest you make an issue in that project. Something like ">=1.7.3 || >=1.8.0-dev" likely works.

**Comment by panuhorsmalahti** [comment]
You can also try to depend on `typescript@next` `tslint@next`.

**Comment by ghpabs** [comment]
Thanks for your help.

I've upgraded it to `tslist@next` which resolves to `3.2.0-dev.1` and also supports `typescript@next`.

The issue now is that `gulp-tslint` requires `tslint@^3 || >=3.1.0-dev`. See error below.

`npm WARN EPEERINVALID gulp-tslint@4.1.1 requires a peer of tslint@^3 || >=3.1.0-dev but none was installed.`

Would you be able to upgrade it to `tslint@^3 || >=3.2.0-dev` instead?

**Comment by panuhorsmalahti** [comment]
Published gulp-tslint 4.1.2 which should fix your issue. It supports ``tslint@3.2.0-dev``. npm is pretty limited in this regard..

**Comment by ghpabs** [comment]
Awesome! Love your work.
